tests/test_food_store_coverage.py

```
@@ -28,6 +28,38 @@ def test_expand_query_none(self):
+        # Test with invalid limit that should trigger exception handling
+        result = search_foods("test", limit="invalid")
+        assert isinstance(result, list)
+
+    def test_search_foods_invalid_offset(self):
+        """Тест search_foods с невалидным offset (строка 48-49)"""
+        # Test with invalid offset that should trigger exception handling
+        result = search_foods("test", offset="invalid")
+        assert isinstance(result, list)
+
+    def test_nutrients_for_invalid_per_g(self):
+        """Тест nutrients_for с невалидным per_g (строка 104-105)"""
```

Ruff (0.14.1)

Docstring contains ambiguous `с` (CYRILLIC SMALL LETTER ES). Did you mean `c` (LATIN SMALL LETTER C)?

(RUF002)